src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -28,14 +28,18 @@ const ActionButtons = ({
           </Link>
         </React.Fragment>
       )}
-      <button
-        type="button"
-        className="btn btn-sm mb-1 actionButtons action-btn-style"
-        id="retireConcept"
-        onClick={() => { showDeleteModal(version_url); }}
-      >
-      Remove
-      </button>
+      {source !== 'Custom' && (
```

Comment:
Shouldn't we have a constant for the above?

Reply:
Okay let me assign it to a constant

Reply:
I've assigned it to a constant `CUSTOM_SOURCE`